youtube_dl/extractor/svt.py

```diff
@@ -180,7 +189,7 @@ def _adjust_title(self, info):
 
     def _extract_by_video_id(self, video_id, webpage=None):
         data = self._download_json(
-            'https://api.svt.se/videoplayer-api/video/%s' % video_id,
+            'https://api.svt.se/video/%s' % video_id,
```

Comment:
This will break some extraction scenarios like `svt:1392003-001A`.

Reply:
Reverting